src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer1 = new Bulldozer("big");
+    private Bulldozer bulldozer2 = new Bulldozer("small");
+
+    private Bulldozer bulldozer3 = new Bulldozer("long");
+    private List list = new ArrayList();
+
```

Comment:
You can do it inside get() method and add machines immediately when creating them.